### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by sanathkr**
*Issue #, if available:*

*Description of changes:*
Updated README to better explain what SAM is, how to get started with SAM, provide quick links to documentation, and also help contributors get started.

For rendered content, see https://github.com/sanathkr/serverless-application-model/blob/readme_improvements/README.md

*Description of how you validated changes:*

*Checklist:*

- [ ] Write/update tests
- [ ] `make pr` passes
- [ ] Update documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,94 @@
-[![Build Status](https://travis-ci.org/awslabs/serverless-application-model.svg?branch=develop)](https://travis-ci.org/awslabs/serverless-application-model)
-[![PyPI version](https://badge.fury.io/py/aws-sam-translator.svg)](https://badge.fury.io/py/aws-sam-translator)
-[![Codecov Test Coverage](https://codecov.io/gh/awslabs/serverless-application-model/branch/master/graphs/badge.svg?style=flat)](https://codecov.io/gh/awslabs/serverless-application-model)
-
-![Logo](aws_sam_introduction.png)
+<p align="center">
```
Comment (brettstack) [inline README.md]: Empty tag? Some sort of hack?

Comment (timoschilling) [inline README.md]: This doesn't make sense.

Comment (sanathkr) [reply README.md]: I think so.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,94 @@
...
+</p>
 
 # AWS Serverless Application Model (AWS SAM)
-You can use SAM to define serverless applications in simple and clean syntax.
 
-This GitHub project is the starting point for AWS SAM. It contains the SAM specification, the code that translates SAM templates into AWS CloudFormation stacks, general information about the model, and examples of common applications.
+![Apache-2.0](https://img.shields.io/github/license/awslabs/serverless-application-model.svg)
+![SAM_CLI release](https://img.shields.io/github/release/awslabs/aws-sam-cli.svg)
```
Comment (keetonian) [inline README.md]: Why is the SAM CLI release version here?

Comment (sanathkr) [reply README.md]: I wanted to add badges for information that customers commonly interact with - it is usually the CLI version.

**Pull request merged**